adafruit_bme680.py

```diff
@@ -45,10 +97,34 @@
 except ImportError:
     pass
 
-__version__ = "0.0.0+auto.0"
+__version__ = "3.4.12"
```

Comment:
This version should stay the way it was. All of our libraries have this placeholder value that gets filled in by the actions when releases are made.

Reply:
yes by all means; was not sure how to do this !!